packages/hooks/src/useAvailableWallets.ts

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { useFetch } from '@deriv/api';
+import { useStore } from '@deriv/stores';
+import useWalletList from './useWalletsList';
+
+const useAvailableWallets = () => {
+    const { client } = useStore();
+    const { accounts, loginid, is_crypto } = client;
+    const { data, ...rest } = useFetch('authorize', {
+        payload: { authorize: accounts[loginid || ''].token },
+        options: { enabled: Boolean(loginid) },
+    });
```

Comment:
@aizad-deriv Seems like this is getting used in different places now, Maybe we can add a `useAuthorize` hook to abstract it and reuse it.

cc: @thisyahlen-deriv

Reply:
I agree for what we have now and for whats more to come